src/VersionControl/GitVersionControl.php

```diff
@@ -42,7 +42,7 @@ public function getStagedFiles()
         foreach ($this->getFiles() as $file) {
             list($status, $relativePath) = explode("\t", $file);
 
-            $fullPath = $base . DIRECTORY_SEPARATOR . $relativePath;
+            $fullPath = trim($base . DIRECTORY_SEPARATOR . $relativePath);
```

Comment:
trim may be described as rtirm+ltrim, and here you don't need ltrim, so why to waste time on doing it?